content/english/Kusto-KC7/activity-3.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Activity 3: New Employee Orientation"
```

Comment:
Since there is no work for the students to do here, we can potentially omit the "Activity" prefix here